hardware/Makefile

```diff
@@ -48,6 +48,13 @@ python          ?= python3
 # Enable tracing
 snitch_trace    ?= 0
 
+# Path to DRAMsyslib
+dramsys_resouces_path ?= $(MEMPOOL_DIR)/hardware/dram_rtl_sim/dramsys_lib/resources
+dramsys_simconfig_path ?= $(dramsys_resouces_path)/simulations/HBM2E-3600.json
+dramsys_lib_path ?= $(MEMPOOL_DIR)/hardware/dram_rtl_sim/dramsys_lib
+questa_args += +DRAMSYS_RES=$(dramsys_resouces_path) +DRAMSYS_CONF=$(dramsys_simconfig_path)
+questa_args += -sv_lib $(dramsys_lib_path)/libDRAMSysLibrary
+
```

Comment:
Does this also work if dram is disabled, or should we only add those questa_args if `dram` is defined?